### DISCUSSION TITLE
Modelleerprincipes mbt 'relaties'

### DISCUSSION THREAD

**Discussion by RiX012** [Geonovum/disgeo-imsor#47] [General]
Ik kon zo snel geen principes vinden over de 'relaties' die in EMSO genoemd worden. Bijvoorbeeld of die ergens uit overgenomen worden en op welk niveau relaties gelegd (mogen) worden. Of dat relaties definiërend zijn of niet.

**Comment by pmaria** [comment]
Welke relaties bedoel je precies?

**Reply by RiX012** [reply]
Deze bijvoorbeeld uit EMSO:
![afbeelding](https://user-images.githubusercontent.com/56293445/119623700-bd4eb580-be08-11eb-92e0-8f5a90e4f905.png)

**Reply by pmaria** [reply]
Ah OK. Ik denk dat het een goed idee is om hier over na te denken. Het heeft ook wat relaties met [U5](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#u5-informatiemodel-is-afgestemd-op-gebruik-door-de-doelgroepen). Hoe dit te verwoorden in een concreet principe weet ik nog niet zo goed. Voorstellen zijn welkom :)

**Reply by RiX012** [reply]
Voorbeelden zou kunnen zijn:
* Dat wanneer relaties geometrisch af te leiden zijn we ze niet opnemen in informatiemodel?
* Dat we bij expliciete relaties uitgaan van de voorgedefinieerde relaties uit een standaard (NEN3610 of NEN2660)?
* Uitgangspunt dat we in principe uitgaan van <<Relatiesoorten>>, en alleen wanneer er echt noodzaak is <<Relatieklasse>> gebruiken.
  * Noodzaak kan nog verschillen: directe eigenschappen van die Relatieklasse of registratiegegevens.

Zal nog eens verder nadenken
